__tests__/ResourceStateBuilder.test.js

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import ResourceStateBuilder from '../src/ResourceStateBuilder'
+import shortid from 'shortid'
+
+const rdf = require('rdf-ext')
+const ParserN3 = require('@rdfjs/parser-n3')
```

Comment:
Did you consider using `n3/lib/N3Parser` instead?  That's what I used here: https://github.com/LD4P/sinopia_editor/pull/808/files#diff-a8ce107dbeb5f2fb8b01abcfc5e7fa02R10 so if we use the same libraries it'll be easier to stitch these together.

Reply:
`n3/lib/N3Parser` is also what we use in `sinopia_acl` so :+1:

Reply:
Switched and added a helper function to Utilities.js for loading.